content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/log-in-chainguard-enforce/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Signing into Chainguard Enforce"
+type: "article"
+description: "How to log into Chainguard Enforce for Kubernetes"
+date: 2022-11-30T11:07:52+02:00
+lastmod: 2022-11-30T11:07:52+02:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "chainguard-enforce-kubernetes"
+weight: 500
+toc: true
+---
+
+> _This document relates to Chainguard Enforce, which you will need access to in order to follow along. You can request access through selecting **Chainguard Enforce for Kubernetes** on our [inquiry form](https://www.chainguard.dev/get-demo?utm_source=docs)._
+
+Chainguard Enforce currently offers login and authentication via three different OpenID Connect identity providers: Google, GitHub, and GitLab. [OpenID Connect](https://openid.net/connect/) (OIDC) provides an identity layer on top of the OAuth 2.0 protocol, and it is integrated with both the Chainguard Enforce Console web browser interface, and `chainctl` on the command line. This document will walk through both of these workflows.
+
+## Signing in through the Chainguard Enforce Console
+
+Once you have access to Chainguard Enforce, you can navigate a web browser to [console.enforce.dev](https://console.enforce.dev). A sign in page will be displayed, similar to the one below. 
+
+![Chainguard Enforce Console sign in page](console-sign-in.png)
+
+From here, you have three options to authenticate into the Console via one of the following OIDC providers: 
+
+* [Google](https://developers.google.com/identity/openid-connect/openid-connect)
+* GitHub
+* [GitLab](https://docs.gitlab.com/ee/integration/openid_connect_provider.html)
+
+**To authenticate with Google** (including through a Google-based email), click the **Google** button, and then select the specific Google account you would like to use to log into [chainguard.dev](https://chainguard.dev). Note that you may need to log into your Google account. 
```

Comment:
I went with bold instead of headers since there is not a lot of text, but we could also make these H3s so that readers can jump into each section